buildScripts/release.groovy

```diff
@@ -77,6 +81,19 @@ pipeline {
                             if (fileExists('tck')) {
                                 sh "find tck -name \"*.jar\" | xargs -I{} scp {} genie.microprofile@projects-storage.eclipse.org:/home/data/httpd/download.eclipse.org/microprofile/staging/${params.module}-${params.releaseVersion}/"
                             }
+                            if (fileExists('tracing/spec')) {
+                                sh "ssh genie.microprofile@projects-storage.eclipse.org mkdir -p /home/data/httpd/download.eclipse.org/microprofile/staging/${params.module}-${params.releaseVersion}"
+                                sh "scp -r tracing/spec/target/generated-docs/* genie.microprofile@projects-storage.eclipse.org:/home/data/httpd/download.eclipse.org/microprofile/staging/${params.module}-${params.releaseVersion}"
```

Comment:
I'm not sure if it's best to copy artifacts from microprofile-telemetry/tracing/* to just microprofile-telemetry. I suppose the telemetry group has a valid case for moving the tracing the spec into a separate module so we should keep the distinction in case they plan to add metrics or logs from telemetry in future.

Reply:
okay, thanks Martin! I have changed the destination to keep the original structure.